.github/workflows/test-api-contract.yaml

```diff
@@ -44,3 +45,8 @@ jobs:
         run: ./scripts/frontend-development/run-contract-test.sh
 
       - run: ./scripts/backend-development/stop-mock-server.sh
+
+      - uses: stefanzweifel/git-auto-commit-action@v4
+        with:
+          file_pattern: "docs/docs/api/openapi.json"
```

Comment:
this ensures will only ever auto-commit for the `docs/docs/api/openapi.json` file to reduce risk of any other files accidentally getting autocommited for whatever reason.